### PR TITLE
Doc update 3.2.1

### PR DESCRIPTION
**Pull request by JaydenLiang**
documentation related update for:
1. feature FAZ integration
2. FOS support 6.4.4

### CODE REVIEW

**Review by jamie-pate — APPROVED** [review]

File: templates/autoscale-existing-vpc.template.yaml [hunk 1]
```diff
@@ -751,7 +751,7 @@ Metadata:
             KeyPairName:
                 default: Key pair name
             FortiGatePskSecret:
-                default: FortiGate PSK secret
+                default: FortiGate PSK secret key
```
Comment (klc178) [inline templates/autoscale-existing-vpc.template.yaml:754]:
```suggestion
                default: FortiGate PSK secret
```

Comment (klc178) [reply templates/autoscale-existing-vpc.template.yaml:754]: After a discussion, the conclusion was to leave it as is.

File: templates/autoscale-main.template.yaml [hunk 2]
```diff
@@ -1495,7 +1495,7 @@ Metadata:
             KeyPairName:
                 default: Key pair name
             FortiGatePskSecret:
-                default: FortiGate PSK secret
+                default: FortiGate PSK secret key
```
Comment (klc178) [inline templates/autoscale-main.template.yaml:1498]:
```suggestion
                default: FortiGate PSK secret
```

File: templates/autoscale-new-vpc.template.yaml [hunk 3]
```diff
@@ -821,7 +821,7 @@ Metadata:
             KeyPairName:
                 default: Key pair name
             FortiGatePskSecret:
-                default: FortiGate PSK secret
+                default: FortiGate PSK secret key
```
Comment (klc178) [inline templates/autoscale-new-vpc.template.yaml:824]:
```suggestion
                default: FortiGate PSK secret
```

File: templates/autoscale-tgw-new-vpc.template.yaml [hunk 4]
```diff
@@ -761,7 +761,7 @@ Metadata:
             KeyPairName:
                 default: Key pair name
             FortiGatePskSecret:
-                default: FortiGate PSK secret
+                default: FortiGate PSK secret key
```
Comment (klc178) [inline templates/autoscale-tgw-new-vpc.template.yaml:764]:
```suggestion
                default: FortiGate PSK secret
```

**Review by klc178 — CHANGES_REQUESTED** [review]

**Review by klc178 — APPROVED** [review]

**Review by klc178 — APPROVED** [review]

**Pull request merged**